### PR TITLE
change picture

### PR DESCRIPTION
**Pull request by YaelShamai**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes:(https://jira-dc.paloaltonetworks.com/browse/CIAC-11482)

## Description
Change SentinalOne integration's picture
## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JasBeilin — APPROVED** [review]
Lets add release notes.

File: Packs/SentinelOne/ReleaseNotes/3_2_29.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### SentinelOne v2
+- Changed the integration's picture
```
Comment (ShirleyDenkberg) [inline Packs/SentinelOne/ReleaseNotes/3_2_29.md:3]:
```suggestion
Changed the integration's picture
```

**Comment by ShirleyDenkberg** [comment]
@JasBeilin Doc review completed.

File: Packs/SentinelOne/Integrations/SentinelOne/SentinelOne_image.png
Comment (JasBeilin) [inline Packs/SentinelOne/Integrations/SentinelOne/SentinelOne_image.png:1]: change to v2 :)

**Review by JasBeilin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**